openmdao/core/system.py

```diff
@@ -3559,20 +3559,18 @@ def _write_table(self, var_type, var_data, hierarchical, print_arrays, all_procs
                     if name not in var_dict:     # If not in the merged dict, add it
                         var_dict[name] = proc_vars[name]
                     else:
-                        # In there already, only need to deal with it if it is a distributed array
-                        # Checking to see if distributed depends on if it is an input or output
-                        if var_type == 'input':
-                            is_distributed = meta[name]['src_indices'] is not None
-                        else:
+                        try:
```

Comment:
This try block is unnecessary.  Just use `allprocs_meta[name]['distributed'] `for all cases.  The 'distributed' entry exists in both `meta` and `allprocs_meta`, but only local variables will be found in `meta`.  `allprocs_meta` will contain all variables from all procs.